index.js

```diff
@@ -153,8 +154,8 @@ function toString(encoding) {
 }
 
 /* Create a message with `reason` at `position`.
- * When an error is passed in as `reason`, copies the
- * stack.  This does not add a message to `messages`. */
+ * When an error is passed in as `reason`, copies the stack.
+ * Errors are not added as a message to `messages`. */
```

Comment:
I guess this line (`Errors are not added as a message to `messages`.`) no longer applies? Could you remove it?